internal/http/services/dataprovider/dataprovider.go

```diff
@@ -75,7 +80,16 @@ func New(m map[string]interface{}, log *zerolog.Logger) (global.Service, error)
 		return nil, err
 	}
 
-	dataTXs, err := getDataTXs(conf, fs)
+	var publisher events.Publisher
+
+	if conf.NatsAddress != "" && conf.NatsClusterID != "" {
```

Comment:
Should It log if it is not configured to use nats? Otherwise admins might not be aware...

Reply:
Agreed :+1: